DESCRIPTION

```diff
@@ -47,7 +45,8 @@ Suggests:
     usethis,
     withr
 Remotes:
-    gsm=Gilead-BioStats/gsm@dev
+    gsm=Gilead-BioStats/gsm@dev,
+    gt=jonthegeek/gt@selection
```

Comment:
this is fine as a temporary patch, but let's make an issue to move `gt` to the imports section as soon as this gets merged into a new `gt` release.

Reply:
Definitely will do!

Reply:
#300